src/main/java/org/folio/circulation/item/service/CirculationItemsService.java

```diff
@@ -0,0 +1,85 @@
+package org.folio.circulation.item.service;
+
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+import java.util.UUID;
+
+import org.folio.circulation.item.domain.entity.Item;
+import org.folio.circulation.item.domain.mapper.CirculationItemsMapper;
+import org.folio.circulation.item.exception.IdMismatchException;
+import org.folio.circulation.item.exception.ResourceAlreadyExistException;
+import org.folio.circulation.item.repository.CirculationItemsRepository;
+import org.folio.rs.domain.dto.CirculationItem;
+import org.folio.spring.exception.NotFoundException;
+import org.springframework.stereotype.Service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@Service
+@RequiredArgsConstructor
+@Log4j2
+public class CirculationItemsService {
+
+  private final CirculationItemsRepository circulationItemsRepository;
+
+  private final CirculationItemsMapper circulationItemsMapper;
+
+  public CirculationItem getCirculationItemById(String id) {
+    var item = getCirculationItemEntityOrThrow(id);
+    return circulationItemsMapper.mapEntityToDto(item);
+  }
+
+  private Item getCirculationItemEntityOrThrow(String id) {
+    return circulationItemsRepository.findById(UUID.fromString(id))
+           .orElseThrow(() -> new NotFoundException(String.format("Circulation Item was not found by id= %s ", id)));
+  }
+
+
+    public CirculationItem createCirculationItem(String circulationItemId, CirculationItem circulationItem) {
+    if(Objects.nonNull(circulationItem.getId()) &&
+            !Objects.equals(circulationItemId, String.valueOf(circulationItem.getId()))) {
```

Comment:
We can use ObjectUtils class for not equal condition .

Reply:
org.apache.commons.commons-lang3 --> ObjectUtils.notEqual bases at equals, which is deprecated since Java 7, when Objects.equals appeared.

So, the approach !Objects.equals is more suitable from this perspective.